docker/main/docker-deploy.yml

```diff
@@ -110,6 +110,43 @@ services:
       - FILES_DIRECTORY=/hydrofabric_data
     entrypoint: ["python3", "-m", "${PYTHON_PACKAGE_NAME_SUBSET_SERVICE:?}"]
     command: ["--files-directory", "/hydrofabric_data", "--port", "${DOCKER_SUBSET_CONTAINER_PORT:-5000}"]
+  data-service:
+    image: ${DOCKER_INTERNAL_REGISTRY:?}/dmod-data-service
+    networks:
+      - requests-net
+    deploy:
+      #mode: global
+      #placement:
+      #  constraints:
```

Comment:
Any reason there is NOT a constraint on this?  Should we create a label `data-node` or something we can pin these to?  Seems like we wouldn't want this on the compute nodes (though we should probably create a `compute-node` label for those as well if we are going that route.

Reply:
I think I deemed a constraint non-essential at the time, and I didn't want to worry about having to properly create and document additional _.env_ variables.  I suppose we could just change these to say `DOCKER_DATASERVICE_DEPLOY_CONSTRAINT_1` and `2` instead of `REQUESTS`, rely on variable-replacement defaults, and just not document them fully yet.  But I'm not sure I wouldn't rather leave that until a time when, as you suggest, we properly refine exactly what constraint configuration we need, here and elsewhere.

Reply:
I went ahead and opened #164 for this, but let me know if you think we need to do something more immediately.

Reply:
That should suffice to track this and develop a more comprehensive strategy.   Worst case scenario, this can be manually tweaked to support a particular deployment for the time being.